AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile

```diff
@@ -1,8 +1,9 @@
 # Stage: build
-FROM node:14 as build
+FROM node:18 as build
 WORKDIR /app
 COPY package*.json /app/
-RUN npm install
+RUN npm install -g npm@10.7.0
```

Comment:
Unsure why we need to pin to a specific npm version

Reply:
Initially experienced issues running it tested again and worked. I have removed that line.